src/components/InventoryTable/hooks/useInventoryExport/helpers.js

```diff
@@ -0,0 +1,32 @@
+import {
+  EXPORT_SERVICE_APPLICATON,
+  EXPORT_SERVICE_RESOURCE,
+} from './constants';
+
+export const buildExportRequestJson = (filters, format) => ({
+  name: 'inventory-export',
+  format,
+  sources: [
+    {
+      application: EXPORT_SERVICE_APPLICATON,
+      resource: EXPORT_SERVICE_RESOURCE,
+      filters,
+    },
+  ],
+});
+
+// TODO Extend the function in fec to allow downloading a link and not just a blob and replace this.
+export const downloadFile = async (
```

Comment:
Can we use the shared function https://github.com/RedHatInsights/frontend-components/blob/1599246a16e70d78401e70628797d5fef24f4fae/packages/utils/src/helpers/helpers.ts#L14

Reply:
At the moment the shared function only supports "downloading" blobs and wouldn't work correctly here. yet. Once we updated the shared function to work by just passing a URL we can use it here as well.

Reply:
Aha, let's keep this in mind and make it shared function later. I am sure that it will be used somewhere in the future.